lib/galaxy/files/templates/examples/production_onedata.yml

```diff
@@ -0,0 +1,30 @@
+- id: onedata
+  version: 0
+  name: Onedata Storage
+  description: |
+    This template allows connecting to Onedata servers.
+
+    Onedata is a global data management system providing a unified data access
+    interface across different storage providers. More information on Onedata
+    can be found in [Onedata's website](https://onedata.org/#/home).
+  variables:
+    onezone_domain:
+      label: Onezone domain
+      type: string
+      help: Domain of the Onezone service (e.g. datahub.egi.eu) to connect to.
+    disable_tls_certificate_validation:
+      label: Disable tls certificate validation?
+      type: boolean
+      help: |
+        Allows connection to Onedata servers that do not present trusted SSL certificates.
+        SHOULD NOT be used unless you really know what you are doing.
+      default: False
+  secrets:
+    access_token:
+      label: REST Access Token
+      help: Your access token, suitable for REST API access in a Oneprovider service.
```

Comment:
Można jeszcze rozważyć do tych helpów access token (2x) podlinkowanie https://onedata.org/#/home/documentation/topic/stable/tokens (ale nie wiem jak to będzie wyglądać)